frontend/lib/analytics/amplitude.ts

```diff
@@ -195,6 +201,12 @@ export function updateAmplitudeUserPropertiesOnSessionChange(
   return true;
 }
 
+export function trackFrontendVersionInAmplitude() {
+  getAmplitude()?.setUserProperties({
```

Comment:
Nice! Can't wait to make an amplitude graph that shows how many people are on each version. Then we can have deprecation parties when the last user is off the old version